src/data/markdown/docs/01 guides/02 Using k6/19 Javascript Compatibility Mode.md

```diff
@@ -117,10 +117,9 @@ $ K6_COMPATIBILITY_MODE=extended k6 run script.js
 
 </CodeGroup>
 
-Transforms scripts to ES5.1 using Babel with ES2015 preset, and loads
-[core.js](https://github.com/zloirock/core-js) v2 polyfills. This is
-the default mode, providing greater JavaScript compatibility, at the
-expense of startup time, RAM usage and performance.
+In case of syntax/parsing error, the script will be transformed using Babel with specific plugins bringing the compatibility to ES2015(ES6)+. This means that features such as classes and arrow functions can be used. This does take some time to transpile and the produced code has slightly different line/column numbers. 
```

Comment:
```suggestion
In case of syntax/parsing errors, the script will be transformed using Babel with specific plugins bringing the compatibility to ES2015(ES6)+. This means that features such as classes and arrow functions can be used. This does take some time to transpile and the produced code has slightly different line/column numbers. 
```